api/leaderboard/leadBoardModel.js

```diff
@@ -24,14 +24,14 @@ const getLeaderBoardData = async () =>{
         const Name = child.Name;
         const WP = child.WritingPoints
         const DP = child.DrawingPoints
-        // const TP = child.Total_Points
+        const TP = child.Total_Points
         const outputNames = output.map(chil => chil.Name)
         if(outputNames.includes(Name)){
             output.forEach(chil =>{
                 if(chil.Name == Name){
                     chil.WritingPoints = WP;
                     chil.DrawingPoints = DP;
-                    chil.Total_Points = WP + DP;
+                    chil.Total_Points = WP + DP + TP;
                 }
```

Comment:
good catch on here. definitely makes the leaderboard more operational/functional/fun now that we're tracking cumulative point totals and not just the weekly scores.